packaging/rpm/flightctl.spec

```diff
@@ -47,9 +47,18 @@ install -m 0755 packaging/greenboot/flightctl-agent-running-check.sh %{buildroot
 cp bin/flightctl-agent %{buildroot}/usr/bin
 cp packaging/must-gather/flightctl-must-gather %{buildroot}/usr/bin
 cp packaging/systemd/flightctl-agent.service %{buildroot}/usr/lib/systemd/system
+bin/flightctl completion bash > flightctl-completion.bash
+install -Dpm 0644 flightctl-completion.bash -t %{buildroot}/%{_datadir}/bash-completion/completions/flightctl-completion.bash
```

Comment:
nice!!! :)

Did you use _datadir directly and not %{bash_completions_dir} etc for any particular reason? (I am wondering if it's not supported in the centos/rhel macros yet, or just an error)

Reply:
I suspect that's the case. %{SHELL_completions_dir}-type macros would not expand resulting in a file not found error.